src/views/Hero/index.view.tsx

```diff
@@ -16,11 +16,17 @@ const HeroView: React.FC<HeroProps> = ({ pageName, title, description }) => {
     <div
       className={cx("HeroView", { HeroView__homeView: pageName === "Home" })}
     >
-      <PostcardStack pageName={pageName} />
-      <EmailSubscriptionForm />
-      Hello from HeroView
-      {title}
-      {description}
+      <div className="HeroView__container">
+        <div className="HeroView__visual">
+          <PostcardStack pageName={pageName} />
+        </div>
+        <div className="HeroView__text">
+          <h1 className="HeroView__titleText">CruzHacks 2021 /</h1>
+          <h1 className="HeroView__titleTextTwo">{title}</h1>
```

Comment:
Awesome! If we're going with this approach, I'd suggest adding a modifier to that `titleText` class that we have up already. 

```suggestion
          <h1 className="HeroView__titleText--secondary">{title}</h1>
```

In the styling we can nest the following together: 
```scss
&__titleText {
    @include typography.title-text;
    &--secondary {
           color: var(--wedgewood__dark);
    }
  }
```

Reply:
Yeah, the properties don't inherit. I think `HeroView__titleText--secondary` looks nicer though so I'll change the class name to that.